pcl_msgs/PolygonMesh mesh
sensor_msgs/PointCloud2 cloud
geometry_msgs/Point reference_point
